documentation/database/timescaledb.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: documentation
+menu: timescaledb
+subtitle: TimescaleDB
+---
+# TimescaleDB
+
+## Supported Versions
+
+- `12`
+- `11`
+
+## Support Level
+
+<table class="table">
+    <tr>
+        <th width="25%">Compatible</th>
+        <td>✅</td>
+    </tr>
+    <tr>
+        <th width="25%">Certified</th>
+        <td>❌</td>
+    </tr>
+    <tr>
+        <th width="25%">Guaranteed</th>
+        <td>❌</td>
+    </tr>
+</table>
+
+Support Level determines the degree of support available for this database ([learn more](/documentation/learnmore/database-support)). 
+
+## Driver
+
+<table class="table">
```

Comment:
any reason we are using html tables in markdown?

Reply:
For consistency with all the existing ones...